KarhooUISDK/Workers/Analytics.swift

```diff
@@ -42,53 +42,53 @@ public enum AnalyticsScreen: Equatable {
     case vehicleTracking
 }
 
-final class KarhooAnalytics: Analytics {
+final public class KarhooAnalytics: Analytics {
```

Comment:
Why does it need to be `public`? If we only need to make Client able to use `KarhooAnalytics` you could add `Client`target membership of the file. This way by UI SDK and Client can treat the file as internal.